playbooks/azure/openshift-cluster/group_vars/all/logging.yml

```diff
@@ -0,0 +1,31 @@
+---
+logrotate_scripts:
+- name: syslog
+  paths:
+  - /var/log/cron
+  - /var/log/maillog
+  - /var/log/messages
+  - /var/log/secure
+  - /var/log/spooler
+  options:
+  - daily
+  - rotate 7
+  - compress
+  - sharedscripts
+  - missingok
+  scripts:
+    postrotate: "/bin/kill -HUP `cat /var/run/syslogd.pid 2> /dev/null` 2> /dev/null || true"
+
+journald_vars_to_replace:
```

Comment:
I don't think the copy/paste is warranted to decrease 8G->6G.  I'm not convinced that 8G is a big problem for us?

Reply:
@jim-minter openshift/openshift-azure#403 seems to indicate that. The `/var` partition is getting full in some cases, it could be that just by rotating the syslogs (which were *not* rotated previously) may fix this issue.